### PR TITLE
add participant

### PR DESCRIPTION
**Pull request by sus5pect**

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Please don't make PR from your `master`. This is not considered as a good practice.
Create a separate branch and then make a PR from it.

File: Data/participants.json [hunk 1]
```diff
@@ -21,12 +21,20 @@
       "facebookLink": "https://www.facebook.com/yk1ax",
       "githubLink": "https://github.com/yk1ax"
     },
-    {
+{
     "name": "Ashutosh Yadav",
-    "email":"yadavashu4488@gmail.com",
+    "email":"yadavashu    4488@gmail.com",
```
Comment (dojutsu-user) [inline Data/participants.json:26]: There shouldn't be spaces in between email address.

File: Data/participants.json [hunk 2]
```diff
@@ -21,12 +21,20 @@
...
      "profilePicture":"https://avatars0.githubusercontent.com/u/55954955?s=460&v=4",
      "facebookLink":"https://www.facebook.com/profile.php?id=100025977280583",
      "githubLink":"https://github.com/yadavashu"
     }
```
Comment (dojutsu-user) [inline Data/participants.json:30]:
```suggestion
    },
```

File: Data/participants.json [hunk 3]
```diff
@@ -21,12 +21,20 @@
...
+    {
+        "name": "Sreeja",
+        "email":"sreejaravula1@gmail.com",
+         "profilePicture":"https://avatars0.githubusercontent.com/u/54639791?s=460&v=4",
+          "facebookLink":"https://www.facebook.com/ravula.sreeja.9",
+            "githubLink":"https://github.com/sus5pect"
+    }
+
```
Comment (dojutsu-user) [inline Data/participants.json:38]:
```suggestion
```

File: Data/participants.json (hunk 1 above)
Comment (dojutsu-user) [inline Data/participants.json:26]:
```suggestion
    "email":"yadavashu4488@gmail.com",
```

File: Data/participants.json [hunk 4]
```diff
@@ -21,12 +21,20 @@
       "facebookLink": "https://www.facebook.com/yk1ax",
       "githubLink": "https://github.com/yk1ax"
     },
-    {
+{
```
Comment (dojutsu-user) [inline Data/participants.json:24]:
```suggestion
    {
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by rishabhshuklax** [comment]
Due to your inactivity, I am closing this PR.

**Pull request closed without merging**